docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -98,7 +98,7 @@ options: [--no-optimization]
 
 ## strapi watch-admin
 
```

Comment:
This does not need to be in this PR. We've only used the CLI.md file to test if hot reload was still working :-)